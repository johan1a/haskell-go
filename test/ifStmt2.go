func main() {
if a == b { c }
}
