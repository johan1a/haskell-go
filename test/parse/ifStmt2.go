package main;
func main() {
if a == b {  }
}
