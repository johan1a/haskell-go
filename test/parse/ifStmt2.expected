SourceFile (Package "main") [TopLevelDecl2 (FunctionDecl2 "main" (Signature1 []) (Block [IfStmt (Ifstmt1 (BinExpr (CondExpr (Eq_ (UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand2 (OperandName1 "a"))))) (UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand2 (OperandName1 "b")))))))) (Block []))]))]
